docs/instructors.yaml

```diff
@@ -173,6 +173,9 @@ ennovytje:
     name: Yvonne Hiltemann
     affiliation: [barharbourinn]
 
+federicodimarco:
+    name: Federico DiMarco
```

Comment:
I've added the new instructor to the instructors file, please check that the name is ok, and we can add their affiliation as well if you let me know :)

Reply:
ah, I see you sent me the info in an email, thanks! will update soon